apps/demos/Demos/Chat/Customization/jQuery/data.js

```diff
@@ -0,0 +1,53 @@
+const getTimestamp = function(date, offsetMinutes = 0) {
+    return date.getTime() + offsetMinutes * 60000;
+}
+
+const date = new Date();
+date.setHours(0, 0, 0, 0);
+
+const currentUser = {
+    id: "c94c0e76-fb49-4b9b-8f07-9f93ed93b4f3",
+    name: "John Doe",
+};
+
+const supportAgent = {
+    id: "d16d1a4c-5c67-4e20-b70e-2991c22747c3",
+    name: "Support Agent",
+    avatarUrl: "https://raw.githubusercontent.com/DevExpress/DevExtreme/refs/heads/24_2/apps/demos/images/petersmith.png",
```

Comment:
You don't need to load this png from github, it's accessible on the site. Please use relative path how it's done in Overview demo